data.sql

```diff
@@ -15,4 +15,51 @@ INSERT INTO animals (name, date_of_birth, escape_attempts,neutered, weight_kg) V
   ('Angemon', '2005-06-12', 1, TRUE, -45),
   ('Boarmon', '2005-06-07', 7, TRUE, 20.4),
   ('Blossom', '1998-10-13', 3, TRUE, 17),
-  ('Ditto', '2022-05-14', 4, TRUE, 22);
\ No newline at end of file
+  ('Ditto', '2022-05-14', 4, TRUE, 22);
+
+  INSERT INTO owners (full_name, age)
+VALUES
+   ('Sam Smith', 34),
+   ('Jennifer Orwell', 19),
+   ('Bob', 45),
+   ('Melody Pond', 77),
+   ('Dean Winchester', 14),
+   ('Jodie Whittaker', 38);
+
+
+INSERT INTO species (name)
+VALUES
+   ('Pokemon'),
+   ('Digimon');
+
+UPDATE animals
+SET species_id = CASE
+    WHEN name LIKE '%mon' THEN (SELECT id FROM species WHERE name = 'Digimon')
+    ELSE (SELECT id FROM species WHERE name = 'Pokemon')
+  END;
```

Comment:
> - [ ] Hello @erickma1, I noticed that you have not yet implemented the second requirement as stated [here](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=Modify%20your%20inserted,animals%20are%20Pokemon), which states that the species_id value must be included if all other animals are Pokemon. Please could you implement this requirement?👍

Reply:
Hi @Johnadibe , I am a little confused, I thought I had implemented that requirement as per the results in the query results attached below.

![species_id](https://github.com/erickma1/vet_clinic/assets/110662611/b901a89a-0a52-4277-8ae5-65ebc7a8da83)

Reply:
Hello @erickma1, Apologies, I quickly saw the Else statement you added to same line. Please you can resubmit for rereview okay👍